cli/src/pcluster/config/json_param_types.py

```diff
@@ -303,6 +303,7 @@ def refresh_compute_resource(self, compute_resource_section):
             # Set gpus according to instance features
             gpus = instance_type_info.gpu_count()
             compute_resource_section.get_param("gpus").value = gpus
+            compute_resource_section.get_param("gpu_type").value = instance_type_info.gpu_type()
```

Comment:
The parameter will still be there in the Json because it is defined in mappings.py? Are you saying we should use an empty string "" instead of "no_gpu_type" as the default?

Reply:
either empty "" or None so that it corresponds to a false value in Python. But feel free to leave it as is if you prefer.